Core/include/Acts/Navigation/DetectorNavigator.hpp

```diff
@@ -33,6 +33,9 @@ namespace Experimental {
 
 class DetectorNavigator {
  public:
+  /// Expose the volume type
+  using VolumeType = DetectorVolume;
+
```

Comment:
That's a left over, I tried to expose the `VolumeType` so that `SteppingLogger` and `Step` such as the `MaterialInteraction` would be templated on it ... but then this would introduce templates all over the examples, so abandoned it - for the material interaction we might think to do that at a later stage. Will remove it.